themes/bootstrap3/less/components/record.less

```diff
@@ -44,6 +44,24 @@
     margin-top: 1rem;
   }
 }
+
+@media (max-width: 500px) {
```

Comment:
Arguably these styles could be part of the previous media query, but I don't think the layout change is needed in the 500-800px range.  The actual 500px number is just a suggestion.